src/mlpack/tests/main_tests/preprocess_scale_test.cpp

```diff
@@ -40,16 +41,14 @@ struct PreprocessScaleTestFixture
   }
 };
 
-BOOST_FIXTURE_TEST_SUITE(PreprocessScaleMainTest,
-                         PreprocessScaleTestFixture);
-
-arma::mat dataset = "-1 -0.5 0 1;"
-                    "2 6 10 18;";
+arma::mat PreprocessScaleTestFixture::dataset = "-1 -0.5 0 1;"
```

Comment:
Is this some automated transition?

Reply:
Not it was colliding with the dataset variable in image_load_test.cpp 

Error was , dataset symbol aldeardy defined

Reply:
duplicate symbol '_dataset' in:

Reply:
Ahh, I see, thanks for the clarification.